internal/pkg/scaffold/crd.go

```diff
@@ -41,6 +39,20 @@ type CRD struct {
 
 	// IsOperatorGo is true when the operator is written in Go.
 	IsOperatorGo bool
+
+	once sync.Once
+	fs   afero.Fs // For testing, ex. afero.NewMemMapFs()
```

Comment:
I'm repeating this pattern from [CSV generation](https://github.com/estroz/operator-sdk/blob/2e6b2e30fe53a323a6e19419059999b734d93f2d/pkg/scaffold/olm-catalog/csv.go#L55-L56) for easier testing. In the future this should be refactored into `Input` so all scaffolds can access a test `afero.Fs`.